src/AbstractOperations/broadcasting_abstract_operations.jl

```diff
@@ -0,0 +1,16 @@
+import Oceananigans.Fields: broadcasted_to_abstract_operation
+
+using Base.Broadcast: Broadcasted
+using Base: identity
+
+const BroadcastedIdentity = Broadcasted{<:Any, <:Any, typeof(identity), <:Any}
```

Comment:
Out of curiousity what would be the difference between this and
```julia
 const BroadcastedIdentity = Broadcasted{A, B, typeof(identity), C} where {A, B, C}
```
?

Reply:
No difference I don't think. I saw the first pattern in `Base.Broadcast` and used it to see how it would feel.